### PR TITLE
chore(ai): prep release

### PR DESCRIPTION
**Pull request by thaddmt**
#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] `yarn test` passes and tests are updated/added
- [ ] PR title and commit messages follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax
- [ ] _If this change should result in a version bump_, [changeset added](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) (This can be done after creating the PR.) This does not apply to changes made to `docs`, `e2e`, `examples`, or other private packages.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-ai/src/components/AIConversation/types.ts [hunk 1]
```diff
@@ -75,8 +75,18 @@ export interface SuggestedPrompt {
   inputText: string;
 }
 
+type JSONSchema4TypeName =
```
Comment (thaddmt) [inline packages/react-ai/src/components/AIConversation/types.ts:78]: Importing @types/json-schema would introduce a lot of outdated dependencies here, for some reason this wasn't an issue with the previous version release but I believe they changed it

**Review by dbanksdesign — APPROVED** [review]

File: packages/react-ai/package.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
     "@aws-amplify/ui-react-core": "3.0.19"
   },
   "devDependencies": {
-    "@aws-amplify/data-schema": "0.0.0-ai-20240815192737",
+    "@aws-amplify/data-schema": "^1.5.0",
```
Comment (calebpollman) [inline packages/react-ai/package.json:55]: Does _@aws-amplify/data-schema_ need to be listed as a peer dep?

File: packages/react-ai/src/components/AIConversation/types.ts (hunk 1 above)
Comment (calebpollman) [reply packages/react-ai/src/components/AIConversation/types.ts:78]: Maybe we can give this a nicer reading name?

File: packages/react-ai/package.json (hunk 2 above)
Comment (thaddmt) [reply packages/react-ai/package.json:55]: Hm actually probably not anymore since we don't need to override it, checking

**Review by dbanksdesign — DISMISSED** [review]

File: packages/react-ai/package.json (hunk 2 above)
Comment (thaddmt) [reply packages/react-ai/package.json]: removed

File: packages/react-ai/src/components/AIConversation/types.ts (hunk 1 above)
Comment (thaddmt) [reply packages/react-ai/src/components/AIConversation/types.ts]: renamed

**Review by dbanksdesign — DISMISSED** [review]

File: .changeset/foo-bar-ai.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@aws-amplify/ui-react-ai": minor
```
Comment (calebpollman) [inline .changeset/foo-bar-ai.md:2]: Just curious, are we using "zero versioning" (e.g. `0.1.0`) until GA?

File: examples/next/pages/ui/components/ai/ai-conversation/amplify_outputs.js [hunk 4]
```diff
@@ -1,3 +1,3 @@
-// import amplifyOutputs from '@environments/ai/gen2/amplify_outputs';
-const amplifyOutputs = {};
+import amplifyOutputs from '@environments/ai/gen2/amplify_outputs';
+// const amplifyOutputs = {};
```
Comment (calebpollman) [inline examples/next/pages/ui/components/ai/ai-conversation/amplify_outputs.js:2]:
```suggestion
```

**Review by calebpollman — APPROVED** [review]
🚢

File: .changeset/foo-bar-ai.md (hunk 3 above)
Comment (thaddmt) [reply .changeset/foo-bar-ai.md:2]: Think that's the idea

Comment (dbanksdesign) [reply .changeset/foo-bar-ai.md:2]: that was my thinking

**Review by dbanksdesign — APPROVED** [review]

File: examples/next/pages/ui/components/ai/ai-conversation/index.page.tsx [hunk 5]
```diff
@@ -10,6 +10,7 @@ import { Authenticator, Card } from '@aws-amplify/ui-react';
 
 const client = generateClient<Schema>({ authMode: 'userPool' });
 const { useAIConversation } = createAIHooks(client);
+console.log(client);
```
Comment (calebpollman) [inline examples/next/pages/ui/components/ai/ai-conversation/index.page.tsx:13]:
```suggestion
```

**Review by calebpollman — APPROVED** [review]

**Pull request merged**